src/accessors/IConfigurationExtend.ts

```diff
@@ -6,6 +8,12 @@ import { ISlashCommandsExtend } from './ISlashCommandsExtend';
  * of your Rocketlet. It is provided during initialization of your Rocketlet
  */
 export interface IConfigurationExtend {
+    /** Accessor for registering the assets which need to be. */
+    readonly assets: IAssetExtend;
+
+    /** Accessor for declaring the contextual bar buttons your Rocketlet provides. */
+    readonly contextualBarButtons: IContextualBarButtonsExtend;
```

Comment:
👍 for the assets being served and registered.

Still wonder whether the ContextualBar should be part of the Configuration interface or a client/ui-interface. But maybe I'm stuck in Meteor thinking.